docs/docsite/rst/guide_aci.rst

```diff
@@ -0,0 +1,298 @@
+
+- `Adam Raffe: Learning ACI <https://adamraffe.com/learning-aci/>`_
+- `Luca Relandini: ACI for dummies <http://lucarelandini.blogspot.be/2015/03/aci-for-dummies.html>`_
+- `Jacob McGill: Automating Cisco ACI with Ansible <https://blogs.cisco.com/developer/automating-cisco-aci-with-ansible-eliminates-repetitive-day-to-day-tasks>`_
+
+
+Using the ACI modules
+---------------------
+The Ansible ACI modules provide a user-friendly interface to managing your ACI environment using Ansible playbooks.
+
+For instance ensuring that a specific tenant exists, is done using the following Ansible task:
+
+.. code-block:: yaml
+
+    - aci_tenant:
+        host: my-apic-1
+        username: admin
+        password: my-password
+    
+        tenant: customer-xyz
+        description: Customer XYZ
+        state: present
+
+A complete list of existing ACI modules is available for `the latest stable release <http://docs.ansible.com/ansible/latest/list_of_network_modules.html#aci>`_ as well as `the current development version <http://docs.ansible.com/ansible/devel/module_docs/list_of_network_modules.html#aci>`_.
+
+Standard module parameters
+..........................
+Every Ansible ACI module accepts the following parameters that influence the module's communication with the APIC REST API:
+
+- `host` -- Hostname or IP address of the APIC
+- `port` -- Port to use for communication *(defaults to `443` for HTTPS, and `80` for HTTP)*
+- `username` -- User name used to log on to the APIC *(defaults to `admin`)*
+- `password` -- Password for `username` to log on to the APIC (using password-based authentication)
+- `private_key` -- Private key for `username` to log on to APIC (using signature-based authentication)
+- `certificate_name` -- Name of the certificate in the ACI Web GUI *(defaults to `private_key` file baseename)*
+- `validate_certs` -- Validate certificate when using HTTPS communication *(defaults to `yes`)*
+- `use_ssl` -- Use HTTPS or HTTP for APIC REST communication *(defaults to `yes`)*
+- `use_proxy` -- Use system proxy settings *(defaults to `yes`)*
+- `timeout` -- Timeout value for socket-level communication
+
+
+ACI authentication
+------------------
+
+Password-based authentication
+.............................
+If you want to logon using a username and password, you can use the following parameters with your ACI modules:
+
+.. code-block:: yaml
+
+    username: admin
+    password: my-password
+
+Password-based authentication is very simple to work with, but it is not the most efficient form of authentication from ACI's point-of-view as it requires a separate login-request and an open session to work. To avoid having your session time-out and requiring another login, you can use the more efficient Signature-based authentication.
+
+**NOTE:** Password-based authentication also may trigger anti-DoS measures in ACI v3.1+ that causes session throttling and results in HTTP 503 errors and login failures. 
+
```

Comment:
Maybe add:
```
.. warning:: Never store passwords in plain text.

The "Vault" feature of Ansible allows you to keep sensitive data such as passwords or keys in encrypted files, rather than as plain text in your playbooks or roles. These vault files can then be distributed or placed in source control. See :doc:`playbooks_vault` for more information.

```

Reply:
Good idea. Done.

Reply:
I also used the `.. note::` syntax for the note.